### PR TITLE
Adds some info about views

### PR DESCRIPTION
**Pull request by kiwicopple**

### CODE REVIEW

File: web/docs/guides/database/tables.mdx [hunk 1]
```diff
@@ -348,8 +336,165 @@ create table salaries (
 );
 ```
 
+## Views
```
Comment (steve-chavez) [inline web/docs/guides/database/tables.mdx:339]: I think we need a mention of VIEWS + RLS, a:

```sql
ALTER VIEW your_view OWNER TO authenticated;
```

Is needed for RLS to work ([ref](https://github.com/supabase/supabase/discussions/901#discussioncomment-500895)). That will change in PG 15.

Comment (kiwicopple) [reply web/docs/guides/database/tables.mdx:339]: ah great, thanks Steve! I just copied this from an old blog. I should actually add RLS to this whole page

**Review by burggraf — APPROVED** [review]

File: web/docs/guides/database/tables.mdx [hunk 2]
```diff
@@ -348,8 +336,171 @@ create table salaries (
...
+
+A View is a convenient shortcut to a query. Creating a view does not involve new tables or data. When run, an underlying query is executed, returning its results to the user.
+
+:::caution
+
+PostgreSQL views do not support Row Level Security. This will be released in PostgreSQL v15. Only expose views to your API if the data is considered public.
```
Comment (kiwicopple) [inline web/docs/guides/database/tables.mdx:345]:
```suggestion
PostgreSQL views are called with the privileges of the role that created them. To enable Row Level Security you should change the owner to the "authenticated" role:
`alter view view_name owner to authenticated;`.
```

File: web/docs/guides/database/tables.mdx (hunk 1 above)
Comment (kiwicopple) [reply web/docs/guides/database/tables.mdx:339]: @steve-chavez - I just added the `caution` below. I didn't actually realise you could enable RLS by changing the owner to "authenticated". Thanks for the tip!

File: web/docs/guides/database/tables.mdx [hunk 3]
```diff
@@ -348,8 +336,172 @@ create table salaries (
...
+create view transcripts as
+    select
+        students.name,
+        students.type,
+        courses.title,
+        courses.code,
+        grades.result
+    from grades
+    left join students on grades.student_id = students.id
+    left join courses on grades.course_id = courses.id;
```
Comment (kiwicopple) [inline web/docs/guides/database/tables.mdx:391]:
```suggestion
    left join courses on grades.course_id = courses.id;
    
alter view transcripts owner to authenticated;
```

**Comment by kiwicopple** [comment]
@steve-chavez - changes made. if you think this is OK, can you please merge?

File: web/docs/guides/database/tables.mdx [hunk 4]
```diff
@@ -340,6 +340,12 @@ create table salaries (
 
 A View is a convenient shortcut to a query. Creating a view does not involve new tables or data. When run, an underlying query is executed, returning its results to the user.
 
+:::caution
+
+PostgreSQL views do not support Row Level Security. This will be released in PostgreSQL v15. Only expose views to your API if the data is considered public.
```
Comment (steve-chavez) [inline web/docs/guides/database/tables.mdx]: > PostgreSQL views do not support Row Level Security

The above is not true. I've been trying to refute that myth in many places.  

Maybe something like:

```suggestion
By default, PostgreSQL views bypass Row Level Security unless you change their owner(see https://github.com/supabase/supabase/discussions/1501). PostgreSQL v15 will have a more intuitive control for this.
```

File: web/docs/guides/database/tables.mdx (hunk 1 above)
Comment (steve-chavez) [reply web/docs/guides/database/tables.mdx:339]: @kiwicopple Added a new comment below, if you agree with the change then this should be good to merge.

File: web/docs/guides/database/tables.mdx [hunk 5]
```diff
@@ -348,8 +336,174 @@ create table salaries (
...
 ```
 
+## Views
+
+A View is a convenient shortcut to a query. Creating a view does not involve new tables or data. When run, an underlying query is executed, returning its results to the user.
+
+:::caution
+
+PostgreSQL views are called with the privileges of the role that created them. To enable Row Level Security you should change the owner to the "authenticated" role:
+`alter view view_name owner to authenticated;`.
```
Comment (kiwicopple) [inline web/docs/guides/database/tables.mdx:346]:
```suggestion
By default, PostgreSQL views bypass Row Level Security unless you change their owner (see https://github.com/supabase/supabase/discussions/901). PostgreSQL v15 (coming soon) will have a more intuitive control for this through [security invoker views](https://www.depesz.com/2022/03/22/waiting-for-postgresql-15-add-support-for-security-invoker-views/) and the previous step won't be needed.
```

**Pull request merged**